lms/templates/admin/organization.html.jinja2

```diff
@@ -45,6 +45,11 @@
             <a class="button is-primary" href="{{ request.route_url("admin.instance.create", _query={"organization_public_id": org.public_id}) }}">
                 Add New LTI 1.1 instance
             </a>
+            {% if org.application_instances %}
```

Comment:
We should do the same in LTIRegistrations that also display application instances inline. Just made a note so I don't forget.